pkg/export/service.go

```diff
@@ -47,12 +49,13 @@ const (
 	NatsAll     = ">"
```

Comment:
This should change

Reply:
messaging package replaces this with # for rabbitmq so its not an issue